### DISCUSSION TITLE
Ameca-擬真表情機器人

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#154] [General]
**新聞內容:** 
CNN報導，「Engineered Arts」研發的最新機器人亞美卡，以和人類互動、溝通以及做出表情為強項設計；創辦人Will Jackson表示，設計擬真機器人的初衷就是要與人類互動，因此特別加強臉部表情功能。
擬真機器人的社會用途相當廣泛，除了能擔任陪伴聊天、放鬆心情的角色，甚至能做到與心理諮商師相當的功能。「Engineered Arts」公司開發Ameca系列機器人已投入260萬美元（約新台幣7200萬元），公司也強調作為人工智慧（AI）測試的機器人平台，功能齊備的Ameca相當適合。

**新聞心得:**
當初在電視上看到這個新聞的時候覺得這家公司能夠把機器人做那麼逼真非常厲害，尤其是整個臉部表情很自然，就很像電影中看到超時空來拯救地球的先進機器人的模樣。雖然不能走動只能擺動四肢，但一樣覺得很神奇！

**新聞來源:**
https://www.ettoday.net/news/20211212/2144266.htm?from=pctaglist

![6057454](https://user-images.githubusercontent.com/90883526/147385096-0711d7bf-7686-420b-bd63-9f6493694a68.jpg)

**Comment by cobb110202008** [comment]
這樣的技術如果能用在陪伴病人及獨居老人，感覺會很有社會意義。

**Comment by pocper1** [comment]
這個機器人的外型，好像電影-全民公敵的機器人XDD

**Comment by briankuo123** [comment]
這個機器人一臉我是誰，我在哪裡的樣子

**Comment by wh20502** [comment]
想到一部電影，在說這世界的人都去睡覺了，在移動的都是機器人就覺得很可怕，會不會哪天機器人取代了我去上課呢？

**Comment by jack90325** [comment]
這項科技進步到最後，搞不好連身邊的人變成機器人都不知道

**Reply by yasklop** [reply]
那確實很恐怖欸

**Comment by ghost** [comment]
但其實這種機器人的臉會讓我覺得很恐怖，就像恐怖谷理論一樣，他接近於真實又不太真實，蠻可怕的

**Comment by handsomeniko** [comment]
不是我要說，這根湯瑪士小火車一樣詭異，半夜看到一定嚇死。

**Comment by Shiba010** [comment]
在未來可能都要懷疑身邊的人可能是機器人了.....

**Comment by lx3211** [comment]
感覺對著這種機器人沒有辦法放鬆心情...

**Comment by xCk27x** [comment]
看這篇報導
感覺機器人還未擁有對獨特情境作出特殊表情的功能
還需要結合自然語言等等的技術
路還很長的樣子~

**Comment by BingRong1217** [comment]
機器人越來越人性化了

**Comment by skyling0299** [comment]
我認為不需要讓機器人擬人化，導致無法辨認或錯認真的蠻可怕的

**Comment by Ben6uo** [comment]
這種機器人通常使用多種傳感器和控制器，有點像人的神經?

**Comment by limitx0** [comment]
很像人但是我覺得好可怕
我完全不會想跟超級擬真的機器人互動

**Comment by Raywei0214** [comment]
由於，人臉似乎是一種巧合，但如果我們從一個更大的角度看待問題，這似乎是一種不可避免的事實。人臉的發生，到底需要如何實現，不人臉的發生，又會如何產生。

**Comment by awkward-willy** [comment]
這樣精妙的機器人真的很讓人驚豔，不過或許是恐怖谷理論的原因，隨著盯著他看的時間變長，總有種說不上來的詭異感。但人臉這樣精妙的情緒表達都能被機器所學習，或許未來能在陪伴、照護等工作發揮效用。
